### ISSUE TITLE
Invalid Junit output from dcgoss

### ISSUE THREAD

**Issue by officerJones** [aelsabbahy/goss#549]
**Describe the bug**
dcgoss produces an unparseable junit xml due to logging to be present in the test results.

**How To Reproduce**
Set output format to unit & run dcgoss
```
export GOSS_OPTS="--format junit"
dcgoss run *service* > testresults.xml
```

**Expected Behavior**
testresults.xml contains a parsable xml in junit format.

**Actual Behavior**
testresults.xml also contains following output:
- Text: *service name*
- docker top output
- Text: Deleting *service name*

**Environment:**
 - Version of goss: v0.3.9

**Comment by aelsabbahy** [comment]
Dgoss gets around this by redirecting information output to stderr.

Seems a similar thing needs to be added to dcgoss.

I'm approving this, feel free to submit a PR. I personally don't maintain user submitted wrapper scripts (dcgoss & kgoss)

**Comment by officerJones** [comment]
I tried to create a pull request for this. 
It's the first time I try to contribute to a project so if anything should be done differently, just let me know.

**Comment by aelsabbahy** [comment]
Closed by #550
